vectors.h

```diff
@@ -1,9 +1,22 @@
 //enter the necessary includes - hint 3 different includes needed - google for which includes need to be added for the C functions used
+#include <stdio>
+#include <math.h>
+#include <stdlib> //includes absolute value for normalize
+//do i need to add a .h to the above files?
 
 typedef struct {
  //enter definition for float fields for x,y,z,length
+ float x;
+ float y;
+ float z;
 } vector;
 
 //header for vector_init fucniton
 vector vector_init(float x, float y, float z);
+
 //header for vector_add function, vector_dot function, vector_normalize function and vector_print function
+vector vector_add(vector vectorArg1, vector vectorArg2);
+vector vector_dot(vector vectorArg1, vector vectorArg2);
+//vector vector_normalize(
+vector vector_print(vector vectorArg1);
```

Comment:
Hi ChadChapman, I missed a few days in class, and now I don't know why I can't see your code, is it mean we are in a group and work together? I am a new member with github so that it make me a little bit confuse about how to work with it, and how to submit the assignment. I am almost done with coding but still wonder how to submit it.